cmd/image-registry-metrics-exporter/main.go

```diff
@@ -44,29 +44,42 @@ func main() {
 	}
 	tags.GenerateMetricsOn()
 
-	bindAddr := flag.String("bind-address", ":8080", "address:port to bind /metrics endpoint to")
+	bindAddrHealth := flag.String("bind-address-health", ":8080", "address:port to bind /health and /readiness endpoints to")
```

Comment:
I this line is too long, I can suggest 

```suggestion
	bindAddrHealth := flag.String("bind-address-health", ":8080", "address:port to bind status endpoints to")
```